tests/legacy-cli/e2e/setup/010-local-publish.ts

```diff
@@ -8,8 +8,8 @@ export default async function() {
     'admin',
     '--',
     'publish',
-    '--versionCheck=false',
-    '--branchCheck=false',
+    '--no-versionCheck',
+    '--no-branchCheck',
     '--registry=http://localhost:4873',
```

Comment:
Now that `--tag` is required,  I think this file also needs to add `--tag latest` to the argument list when not a prerelease.  `--tag next` is already added in the prerelease case on line 18.

Reply:
Done.